### PR TITLE
Update enterprise feature visible metadata fields and summary text

### PR DESCRIPTION
**Pull request by yanndago**
* Update summary help text
* Ensure no new fields are added to the enterprise views by separating functions that render enterprise items from normal features
* Hide launch bug and breaking changes from enterprise feature pages

### CODE REVIEW

**Comment by yanndago** [comment]
Screenshot 2023-05-04 at 2 34 43 PM
Screenshot 2023-05-04 at 2 32 00 PM
Screenshot 2023-05-04 at 2 31 36 PM
Screenshot 2023-05-04 at 2 31 36 PM
Screenshot 2023-05-04 at 2 32 00 PM
Screenshot 2023-05-04 at 2 34 43 PM
253-2bf28694-e6e2-43d5-a5cc-b3b6641c516a.png">
Screenshot 2023-05-04 at 2 31 10 PM

**Comment by yanndago** [comment]
Screenshot 2023-05-04 at 2 31 10 PM
Screenshot 2023-05-04 at 2 31 36 PM
Screenshot 2023-05-04 at 2 32 00 PM
Screenshot 2023-05-04 at 2 34 43 PM

File: client-src/elements/chromedash-form-field.js [hunk 1]
```diff
@@ -197,8 +197,9 @@ export class ChromedashFormField extends LitElement {
     const helpText = this.forEnterprise && this.fieldProps.enterprise_help_text ?
       this.fieldProps.enterprise_help_text :
       this.fieldProps.help_text;
-    const extraHelpText = this.fieldProps.extra_help;
-
+    const extraHelpText = this.forEnterprise ?
```
Comment (jrobbins) [inline client-src/elements/chromedash-form-field.js:200]: Would it be better to show the regular extra_help if enterprise_extra_help is not defined?
Something like 
```
const extraHelpText = this.forEnterprise && 
     this.fieldProps.enterprise_extra_help ||
       this.fieldProps.extra_help;
```

**Review by jrobbins — APPROVED** [review]

File: client-src/elements/chromedash-form-field.js (hunk 1 above)
Comment (yanndago) [reply client-src/elements/chromedash-form-field.js]: Hmm yes, I will do that. Updated the logic for the cases where we want to hide the help text for enterprises.

**Pull request merged**